code/modules/materials/material_recipes.dm

```diff
@@ -162,4 +162,5 @@
 		))
 
 /material/snow/generate_recipes()
-	return // Snowmen and snowballs may come here later.
\ No newline at end of file
+	..()

```

Comment:
Did you test this? Because the only thing I want to be craft-able with snow, for the time being, is snowballs.

Reply:
Ah, alright then. It makes things like Necklaces craftable with snow (logic), so I'll cut that out.

Reply:
@Anewbe not actually much I can do about it making the base craftables like necklaces etc..

Reply:
Try something like

/material/snow/generate_recipes()
->recipes = list()
->recipes += new/datum/stack_recipe("snowball", /obj/item/weapon/material/snow/snowball, 1, time = 10)